core/ecschema-locaters/src/test/assets/JSON/AecUnits.ecschema.json

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
I thought we were going to load the test schemas from the schema packages so we did not need to add so much test data to the repo?

Reply:
I was unable to locate the .json files within the schema packages. Would it be advisable to convert the XML files to JSON for testing purposes, or are there alternative options that may be more suitable?

Reply:
Switched to using npm packages for testing now, and as a result, removed the test data.